test/e2e_node/privileged_test.go

```diff
@@ -151,7 +150,6 @@ func (config *PrivilegedPodTestConfig) dialFromContainer(containerIP string, con
 	var output map[string]string
 	err = json.Unmarshal([]byte(stdout), &output)
 	Expect(err).NotTo(HaveOccurred(), fmt.Sprintf("Could not unmarshal curl response: %s", stdout))
-	glog.Infof("Deserialized output is %v", output)
```

Comment:
I agree in general that we should keep the output to the minimum, but fwiw, comparing the logs of a passed test against those of a failed test is helpful at times. I guess we won't need that if every test is well-written  :)